client/my-sites/earn/memberships/index.jsx

```diff
@@ -556,9 +560,7 @@ class MembershipsSection extends Component {
 			return this.renderOnboarding(
 				<Notice
 					status="is-warning"
-					text={ this.props.translate(
-						'Only site administrators can edit Recurring Payments settings.'
-					) }
+					text={ this.props.translate( 'Only site administrators can edit Payments settings.' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.